morph.js

```diff
@@ -5,9 +5,16 @@ module.exports = morph
 
 // diff elements and apply the resulting patch to the old node
 // todo (yw): copy over events
-// todo (yw): investigate what else to copy over
-// (obj, obj) -> null
+// todo (yw): investigate what else to copy over // (obj, obj) -> null
```

Comment:
think we can ditch the type notation if you don't like it (: - I'm on the edge if we should use it anyway; def not merging it like this haha